include/experimental/group/gemm/impl/int4_dequantize_xe.hpp

```diff
@@ -176,12 +190,20 @@ class gemm_t<
 
   // note: plane format, row-major
   // note: 4bit x 2, row-major
-  using matB_tile_desc_t = subgroup::tile_desc_t<
-      tile_size_x_b / pack_ratio,
-      tile_size_y_b,
-      block_size_x_b / pack_ratio,
-      block_size_y_b,
-      reg_layout::tiled>;
+  using matB_tile_desc_t = std::conditional_t<
+      is_col_major_b,
```

Comment:
do we need to support col_major too? there will be double size of templates expansion and slower compilation especially in IPEX.